store/v2alpha1/multi/params.go

```diff
@@ -11,10 +12,11 @@ import (
 // pruning with PruneDefault, no listeners and no tracer.
 func DefaultStoreParams() StoreParams {
 	return StoreParams{
-		Pruning:          pruningtypes.NewPruningOptions(pruningtypes.PruningDefault),
-		SchemaBuilder:    newSchemaBuilder(),
-		storeKeys:        storeKeys{},
-		traceListenMixin: newTraceListenMixin(),
+		Pruning:                   pruningtypes.NewPruningOptions(pruningtypes.PruningDefault),
+		SchemaBuilder:             newSchemaBuilder(),
+		storeKeys:                 storeKeys{},
+		traceListenMixin:          newTraceListenMixin(),
+		substoreTraceListenMixins: make(map[types.StoreKey]*traceListenMixin),
```

Comment:
With the `substoreTraceListenMixins`, the `traceListenMixin` seems a bit redundant. Or is it still required?

Reply:
I view `traceListenMixin` as a tracer for `Multistore` and the `substoreTraceListenMixins` as tracers for each individual `subStore`

Reply:
Ah OK. We can later make this a bit more explicit by renaming the variables or at least adding comments. Once the PR is feature complete.